package.json

```diff
@@ -9,10 +9,12 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "framer-motion": "^12.6.0",
     "next": "^15.1.7",
     "react": "^19.0.0",
     "react-click-outside": "^3.0.1",
     "react-dom": "^19.0.0",
+    "react-intersection-observer": "^9.16.0",
```

Comment:
Is this being used?

Reply:
No, good catch. As I'm doing a rewrite of the site, is there a good way to update package.json to remove unused dependencies or should I go through the project carefully and manually remove unused dependencies?

Reply:
[This little script](https://gist.github.com/coltenkrauter/fbd6cbddcadd425ba3ffd4f69a9324d5) should do the trick.

Reply:
Thank you! I just used this, this is great!

Reply:
Please star the script :)

Reply:
You got it!